locales/en-US/app.ftl

```diff
@@ -78,6 +78,16 @@ CallNodeContextMenu--transform-focus-subtree = Focus on subtree only
         specific part of the call tree. It pulls out a branch of the call tree,
         however it only does it for that single call node. All other calls
         of the function are ignored.
+CallNodeContextMenu--transform-focus-category = Focus on category
```

Comment:
Suggestion: provide the name of the category that will be focused.

Reply:
No, it would the only transform that would contain such a term.

Reply:
That is not true, Johannes.

For example:
```
CallNodeContextMenu--transform-collapse-resource =
    Collapse <strong>{ $nameForResource }</strong>
    .title =
        Collapsing a resource will flatten out all the calls to that
        resource into a single collapsed call node.
```

Reply:
Oh, sorry, I must have overlooked it. Fix it soon.

Reply:
<img width="695" alt="image" src="https://user-images.githubusercontent.com/490655/198568484-249b9da4-9b54-46b4-91f3-daeb4cce1c2d.png">